zerver/actions/users.py

```diff
@@ -525,8 +701,10 @@ def do_deactivate_user(
             event_time=event_time,
             extra_data={
                 RealmAuditLog.ROLE_COUNT: realm_user_count_by_role(user_profile.realm),
+                "removed_permissions": removed_permissions_audit_log_data,
```

Comment:
Not sure if this field name is fine. The value here is a list of objects with one object per setting. For stream and groups, each setting for a stream or group is a separate object.